src/ansys/sherlock/core/types/project_types.py

```diff
@@ -72,8 +72,8 @@ def __init__(
         board_bounds: BoardBounds,
         coordinate_units: str,
         image_bounds: ImageBounds,
-        legend_bounds: LegendBounds,
-        legend_orientation: LegendOrientation,
+        legend_bounds: "LegendBounds",
+        legend_orientation: "LegendOrientation",
```

Comment:
Adding quotes allows the class name to be resolved even though the class is defined after this method.